### ISSUE TITLE
Increase contrast color for "Loans" links

### ISSUE THREAD

**Issue by tclayton33** [emory-libraries/librarysearch-enhance#48]
The color of the links listing user's "loans", when viewing them from My Library Card - Loans tab, does not pass the contrast test for accessibility standards. Please darken the color to increase contrast and pass  WCAG AA standards. See https://webaim.org/resources/contrastchecker/

Color options: **#002878 (color used in menu bar under "My Library Card")**; #004990 (color used in top menu bar for Library Search); #325C7B (blends/slightly darker variation of color currently used for "renew" and "renew all" links

Current color of title links:
![Image](https://github.com/user-attachments/assets/906d18df-a636-490e-b635-2d0787ee9a45)

Proposed color of title links [image needs updating w/ actual preferred color]:
<img width="1065" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBejJHQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b3241abf4932c6903fcbfaa5c2d94aa299914751/Link%20color%20%23325C7B.png" alt="Link color #325C7B.png" />

The following was sent to Library Search Feedback: I noticed today when accessing the “Loans” page through my library account that the hyperlink text is a light blue color, #d0defd (screenshot attached). While I can manage, or change the color temporarily through the inspector tool, others may not as easily. Is there any chance you can pass this email to someone that can update it to a darker hue? From what I can tell, it’s the “prm-loans h3 a” element.

**Comment by libah** [comment]
I increased the contrast color.<img width="1321" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzdDQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--81ddb74b1614f12efe66f07b24bfc392e7161728/loansMonosnap%20Emory%20University%20Libraries%20-%20My%20Library%20Card%20-%20Loans%202025-04-22%2009-47-17.png" alt="loansMonosnap Emory University Libraries - My Library Card - Loans 2025-04-22 09-47-17.png" />

**Comment by libah** [comment]
To test:
1. Go to  https://emory-psb.primo.exlibrisgroup.com/discovery/account?vid=01GALI_EMORY:servicescopy&section=loans&lang=en
2. login as Library Staff:  libkey1 testingisfun!2024

**Comment by tclayton33** [comment]
@libah Something that changed also flipped the hover background when you hover over the "Renew" buttons. That text should be white when hovering, but it is now blue (so seeing blue on blue).

@tclayton33 working on it. You may see some wonky things while I'm doing this.

**Comment by libah** [comment]
@tclayton33 Fixed the "Renew" buttons. However, now I see the pin icon in the loans section, which I don't see in production. I'm investigating. https://emory-psb.primo.exlibrisgroup.com/discovery/account?vid=01GALI_EMORY:servicescopy&section=loans&lang=en

**Comment by libah** [comment]
pin icon is gone.
